.travis.yml

```diff
@@ -45,6 +53,14 @@ matrix:
                   latest_file="$(basename $latest_file .rootfs)"
                   # Find all files with that prefix and copy them to our cache dir
                   sudo find /var/snap/lxd/common/lxd/images/ -name $latest_file* -print -exec cp {} lxd_images/ \;
+              - |
+                  # If a schroot exists (i.e. we didn't fail before its
+                  # creation), tar it up (to preserve ownership/permissions)
+                  # and move it into the cached dir; no need to compress it
+                  # because Travis will do that anyway
+                  if [ -e /var/lib/schroot/chroots/xenial-amd64 ]; then
+                    sudo tar cf chroots/xenial-amd64.tar -C /var/lib/schroot/chroots/xenial-amd64 .
```

Comment:
The comments above mention preserving permission/ownership;  I suggest using --preserve here for clarity; though manpage on tar suggests that for superuser (sudo) it is the default.  It can't hurt to be explicit though.

Reply:
it may not matter to our tests but ideally we preserve the debootstrap as much as possible in case that it does later on.
I suggest adding to the tar command:
--sparse
--xattrs --xattrs-include=*